openwisp_utils/admin_theme/email.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+
+from . import settings as app_settings
+
+
+def send_email(subject, body, recipients, extra_context={}):
+
+    mail = EmailMultiAlternatives(
+        subject=subject,
+        body=strip_tags(body),
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        to=recipients,
+    )
+    context = dict(
+        subject=subject,
+        message=body,
+        logo_url=app_settings.OPENWISP_EMAIL_LOGO,
+        call_to_action_text='Find out more',
```

Comment:
Are we sure we want to leave this here if there's no default URL? It doesn't make much sense to me. Why do we want to leave this here? Is there a specific reason?

**BTW keep in mind user facing strings must always be flagged as translatable**.

Reply:
I left this as if a user wants to give a url only, then we can use it as default.
However, it is not used until URL is passed.
Removing it now.